aclk/agent_cloud_link.c

```diff
@@ -1400,18 +868,25 @@ void *aclk_main(void *ptr)
         }
     }
 
-    aclk_stats_enabled = appconfig_get_boolean(&cloud_config, CONFIG_SECTION_GLOBAL, "statistics", CONFIG_BOOLEAN_YES);
+    query_threads.count = config_get_number(CONFIG_SECTION_CLOUD, "query thread count", 2);
+    if(query_threads.count < 1) {
+        error("You need at least one query thread. Overriding configured setting of \"%d\"", query_threads.count);
+        query_threads.count = 1;
```

Comment:
The appconfig_set_number should change to config_set_number to go to netdata conf and the correct section